public/locales/en/panel-nightsky.json

```
@@ -0,0 +1,249 @@
+{
+  "tab-headers": {
+    "markings": "Markings",
+    "time": "Time",
+    "location": "Location / View",
+    "stars": "Stars",
+    "solar-system": "Solar System",
+    "sun": "Sun"
+  },
+  "markings": {
+    "button-labels": {
+      "hide-all": "Hide All",
+      "grid": "Grid",
+      "labels": "Labels",
+      "line_one": "Line",
+      "line_other": "Lines"
+    },
+    "alt-az": {
+      "title": "Alt/Az",
+      "buttons": {
+        "meridian": "Meridian",
+        "zenith": "Zenith"
+      }
+    },
+    "ecliptic": {
+      "title": "Ecliptic",
+      "buttons": {
+        "band": "Band"
+      }
+    },
+    "equitorial": {
+      "title": "Equitorial"
+    },
+    "constellations": {
+      "title": "Constellations",
+      "buttons": {
+        "art": "Art"
+      },
+      "aria-labels": {
+        "lines": "Show constellation lines",
+        "labels": "Show constellation labels",
+        "art": "Show constellation art"
+      }
+    },
+    "cardinal-directions": {
+      "title": "Cardinal Directions",
+      "buttons": {
+        "small": "Small",
+        "large": "Large",
+        "marks": "Marks"
+      },
+      "aria-labels": {
+        "small": "Show small cardinal directions",
+        "large": "Show large cardianl directions",
```

```suggestion
        "small": "Show cardinal directions (small)",
        "large": "Show cardinal directions (large)",
```

?
Since the size seems more like an attribute of the cardinal directions rather than being different datasets?